pkgs/applications/misc/synergy/default.nix

```diff
@@ -27,12 +27,12 @@
 , avahi-compat
 
   # MacOS / darwin
-, darwin
 , ApplicationServices
 , Carbon
 , Cocoa
 , CoreServices
 , ScreenSaver
+, UserNotifications ? null
```

Comment:
```suggestion
, UserNotifications
```

Reply:
Done in commit 7bf7cb5a576fc72ccca5bb217fbb625e0c1a27af.